modules/vocabularies/src/main/java/io/uhndata/cards/vocabularies/internal/NCITFlatIndexer.java

```diff
@@ -114,7 +114,8 @@ private void createTermNodes(final File source, Map<String, String[]> parentsMap
     {
         // The NCIT source is an unquoted tab-delimited file
         // We need withQuote(null) to keep all quotes as part of the text, and not interpreted as special chars
-        try (CSVParser csvParser = CSVParser.parse(source, DEFAULT_CHARSET, CSVFormat.TDF.withQuote(null))) {
+        try (CSVParser csvParser = CSVParser.parse(source, DEFAULT_CHARSET,
+                CSVFormat.Builder.create().setQuote(null).build())) {
```

Comment:
This is not the same as before, `Builder.create()` uses the default CSV configuration, not the `TDF` one. You should do:

```suggestion
                CSVFormat.TDF.builder().setQuote(null).build())) {
```

Same in all the other places where you need a CSV format.